synapse/models/orgs.py

```diff
@@ -354,6 +354,9 @@ def getModelDefs(self):
                         'doc': 'An array of NAICS codes that map to the industry.'}),
                     ('isic', ('array', {'type': 'ou:isic', 'uniq': True, 'split': ','}), {
                         'doc': 'An array of ISIC codes that map to the industry.'}),
+                    ('desc', ('str', {}), {
+                        'doc': 'A description of the industry.',
```

Comment:
Should this have `'disp': {'hint': 'text'}` on it?

Reply:
OK.